portal-impl/src/com/liferay/portal/servlet/ComboServlet.java

```diff
@@ -438,6 +452,8 @@ protected boolean validateModuleExtension(String moduleName)
 		return validModuleExtension;
 	}
 
+	private static final String _NO_SUCH_ENTITY_WITH_KEY = "No Portlet exists with the key {";
```

Comment:
The message should handle on properties file

Reply:
I fixed the issue based on your suggestion